src/clr-angular/wizard/wizard.spec.ts

```diff
@@ -29,6 +29,7 @@ export default function(): void {
           wizard = context.clarityDirective;
           component = context.hostComponent;
           context.detectChanges();
+          context.detectChanges();
```

Comment:
I manage to pass all test without the second `detectChanges`

```bash
Finished in 6 mins 35.946 secs / 6 mins 24.362 secs @ 13:40:33 GMT+0200 (Eastern European Standard Time)

SUMMARY:
✔ 3087 tests completed
ℹ 79 tests skipped
TOTAL: 3087 SUCCESS
```

```diff
--- a/src/clr-angular/wizard/wizard.spec.ts
+++ b/src/clr-angular/wizard/wizard.spec.ts
@@ -29,7 +29,6 @@ export default function(): void {
           wizard = context.clarityDirective;
           component = context.hostComponent;
           context.detectChanges();
-          context.detectChanges();
         });

         describe('open', () => {
```

Reply:
I'm not sure how this change got in there, its not related to this PR. I wIll remove.